FWCore/ParameterSet/python/Mixins.py

```diff
@@ -711,7 +711,7 @@ class __DummyModule(object):
                 def __init__(self):
                     self.tLPTest = tLPTest
                     self.tLPTestType = tLPTestType
-            p = tLPTest("MyType",** { "a"+str(x): tLPTestType(x) for x in xrange(0,300) } )
+            p = tLPTest("MyType",** { "a"+str(x): [tLPTestType(x) for x in xrange(0,300)] } )
```

Comment:
These are not the same thing. The first creates a dictionary with 300 entries, the second creates a dictionary with one entry.